package.json

```diff
@@ -71,6 +73,7 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-logical-assignment-operators": "^7.12.1",
+    "@babel/preset-typescript": "^7.13.0",
```

Comment:
> installed countless packages to solve endless failing test

this is what i found in a commit message, but i see just one package that was installed here?

Reply:
I'm sorry this message is not accurate. I have done a bunch of changes and lost track of what was deleted and what stayed . I would be happy to squash the first commit message with the second